app/src/main/res/drawable/shape.xml

```diff
@@ -0,0 +1,9 @@
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <solid android:color="#F5F5F5"/>
+    <corners android:radius="4dp"/>
+    <padding
+        android:left="4dp"
+        android:top="4dp"
+        android:right="4dp"
+        android:bottom="4dp"/>
+</shape>
```

Comment:
I created this drawable as the background for a card Right now it just has a background color and some rounded corners.